gatling-http/src/main/scala/io/gatling/http/request/builder/HttpRequestBuilder.scala

```diff
@@ -128,9 +128,9 @@ final case class HttpRequestBuilder(commonAttributes: CommonAttributes, httpAttr
   def multivaluedFormParam(key: Expression[String], values: Expression[Seq[Any]]): HttpRequestBuilder = formParam(MultivaluedParam(key, values))
 
   def formParamSeq(seq: Seq[(String, Any)]): HttpRequestBuilder = formParamSeq(seq2SeqExpression(seq))
-  def formParamSeq(seq: Expression[Seq[(String, Any)]]): HttpRequestBuilder = formParam(ParamSeq(seq))
+  def formParamSeq(seq: Expression[Seq[(Any, Any)]]): HttpRequestBuilder = formParam(ParamSeq(seq))
 
-  def formParamMap(map: Map[String, Any]): HttpRequestBuilder = formParamSeq(map2SeqExpression(map))
+  def formParamMap(map: Map[String, String]): HttpRequestBuilder = formParamSeq(map2SeqExpression(map))
```

Comment:
This is wrong: you're losing the ability to pass `formParamMap(Map("foo" -> 1))`.